e2e/playwright/tests/create_legalhold.spec.ts

```diff
@@ -0,0 +1,25 @@
+import {test} from '@e2e-support/test_fixture';
+import{ CreatePluginPage} from '../pages/CreatePluginPage';
+
+
+let createPluginPage : CreatePluginPage;
+
+test('create a legal hold successfully', async ({ pw, pages }) => {
+  const { adminUser, adminClient, user } = await pw.initSetup();
+  const { page } = await pw.testBrowser.login(adminUser);
+
+  await adminClient.enablePlugin('com.mattermost.plugin-legal-hold');
+
+  const systemConsolePage = new pages.SystemConsolePage(page);
+  await systemConsolePage.goto();
+  await systemConsolePage.toBeVisible();
+  
+  await createPluginPage.clickLegalHoldPlugin();
+  await createPluginPage.clickCreateNewButton();
+  await createPluginPage.enterLegalHoldName('Sample Legal Hold');
+  await createPluginPage.selectUsername(user.username);
+  await createPluginPage.enterStartDate('2024-10-28');
+  await createPluginPage.clickLegalHoldButton();
+  await createPluginPage.verifyModalIsNotVisible();
+
+});
```

Comment:
Add comments according to test steps and/or verifications so it's clear for non-technical person to see what it's all about.

Reply:
Btw, this test is not complete. To verify that the legal hold has been created successfully, it should not only check that the modal is hidden but should also check that the created legal hold is on the list and matches:
- name
- start / end date (where in this case, end date is "Never")
- number of users

Reply:
i was thinking of creating a separate tests for that but i will update accordingly